app/src/main/java/org/sopt/and/MyActivity.kt

```diff
@@ -0,0 +1,192 @@
+package org.sopt.and
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Notifications
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material.icons.outlined.Warning
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            ANDANDROIDTheme {
+                // Intent로 전달된 이메일 데이터 받기
+                val email = intent.getStringExtra("email") ?: "프로필 1님"
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    MyPageScreen(modifier = Modifier.padding(innerPadding), email = email)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MyPageScreen(modifier: Modifier = Modifier, email: String) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(100.dp)
+                .background(Color.DarkGray)
+                .padding(10.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Row(
+                modifier = Modifier,
+                horizontalArrangement = Arrangement.Start,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Box(
+                    modifier = Modifier
+                        .size(60.dp)
+                        .background(Color.Cyan, shape = CircleShape)
+                    // 추후 사진으로 변경 필요
```

Comment:
```suggestion
                    // TODO: 사진으로 변경
```
TODO 주석도 존재해요! (색깔도 바껴서 이쁨 ㅋㅋ)